src/transformers/jest-globals-import.ts

```diff
@@ -0,0 +1,92 @@
+import fs from 'fs'
+import type { ImportSpecifier, JSCodeshift } from 'jscodeshift'
+import path from 'path'
+
+import { findImports, removeRequireAndImport } from '../utils/imports'
+
+const jestGlobals = new Set<string>()
+
+const jestGlobalsPath = path.join(
```

Comment:
This import looks a bit funky. Are we importing a type definition file? I'm also pretty sure this will not work in various package system as we assume where dependencies are placed. We should import this like any other third parties.

Reply:
this is definitely the funkiest part.  yes, this is reading a type file.  the point is to dynamically parse the exported members.  yes, it is possibly risky.

we can freeze the list of things exported from jest, it'll just be a little behind / not update automatically.  that could be the right set of tradeoffs for this library (rather than a more homogenous environment).

do you want me to do that?  I think there's already a list of jest-y things in `jest-codemods`, right?

Reply:
Hardcoding them here should be good enough. 

Bonus point for adding a unit test that verifies that the hardcoded list is up to date with Jest.